admin/query/src/main/java/org/codice/ditto/replication/admin/query/sites/persist/DeleteReplicationSite.java

```diff
@@ -60,8 +62,12 @@ public void validate() {
       return;
     }
 
-    ListField<ReplicationField> repFields = replicationUtils.getReplications(true);
     String idToDelete = id.getValue();
+    if (!replicationUtils.siteIdExists(idToDelete)) {
+      addErrorMessage(DefaultMessages.noExistingConfigError());
+    }
```

Comment:
replicationUtils.deletesite() returns a boolean based on whether the id was found or not. You could also argue that if the site doesn't exist then it was probably already deleted and the user doesn't need to worry about it.

Reply:
I agree with @kcover here. If the site did not exist to start with then the end result is the same as if it did and therefore it shouldn't be an error but more of a no-op

Reply:
The UI can treat it as a noop, but I think from and API perspective it would be important to know if the operation actually did something.